app/javascript/css/colors.scss

```diff
@@ -40,6 +40,13 @@
   --primary-color: #3D5D6F;
   --primary-color-dark: #314A59;
   --primary-color-darker: #253843;
+  --default-font-color: #323130;
+  --placeholder-color: #676767;
+  --placeholder-font-weight: 400;
```

Comment:
might be more appropirate in the me or _fonts scss file, since it isnt really a color